low-level/framework/base/sspl_constants.py

```diff
@@ -71,30 +76,37 @@
 CONSUL_PORT = '8500'
 
 # TODO Keep only constants in this file.
-# other values(configs) should come from cofig.
-# Check if SSPL is configured
-if os.path.exists(SSPL_CONFIGURED):
-    try:
-        config = configparser.ConfigParser()
-        config.read(file_store_config_path)
-        storage_type = config['STORAGE_ENCLOSURE']['type']
-        server_type = config['SYSTEM_INFORMATION']['type']
-        cluster_id = config['SYSTEM_INFORMATION']['cluster_id']
-        node_id = config['SYSTEM_INFORMATION']['node_id']
-        node_key_id = config['SYSTEM_INFORMATION']['salt_minion_id']
-        CONSUL_HOST = config['DATASTORE']['consul_host']
-        CONSUL_PORT = config['DATASTORE']['consul_port']
-    except Exception as err:
-        print(f'sspl_constants : Failed to read from {file_store_config_path} due to error - {err}')
-# If not configured, use salt interface
-else:
+# other values(configs) should come from config.
+
+# TODO: Below finding machine id/node_key_id code will be replaced by PR #281 [EO-16515]
+try:
```

Comment:
we still adding code to read dynamic cfg in a file we call sspl_constants.py
This change doesnt really to this PR, so its fine. Though need to change this approach soon

Reply:
Yes.